unicoremc/models.py

```diff
@@ -396,6 +399,20 @@ def create_webhook(self, access_token):
             raise exceptions.AccessTokenRequiredException(
                 'access_token is required')
 
+    def create_unicore_distribute_repo(self):
+        post_data = {
+            "repo_url": self.repo_git_url
```

Comment:
You need to add the "repo_name" field here so that it corresponds to the repo name used in the production.ini repo url.